### PR TITLE
[Week 2] How-to-guide Overview

### PR DESCRIPTION
**Pull request by abhishek-iiit**
Added Different heading in the overview section of how-to-guide
Screenshot from 2021-06-12 17-47-38
Screenshot from 2021-06-12 17-47-51

### CODE REVIEW

File: docusaurus/docs/howto/overview.mdx [hunk 1]
```diff
@@ -1,38 +1,71 @@
...
 slug: /howto/
-title: Wechaty How-to guides overview
+title: How-to guides
 sidebar_label: 'How-to Guides: Overview'
 ---
 
 <!-- how-to/cookbook - Explains how to accomplish a specific task. -->
 
-How-to guides are recipes. They guide you through the steps involved in addressing key problems and use-cases. They are more advanced than tutorials and assume some knowledge of how Wechaty works.
+How-to guides are recipes and they guide you through the steps involved in addressing key problems and use-cases.
```
Comment (proudofsimin) [inline docusaurus/docs/howto/overview.mdx]: How-to guides are like recipes when you are cooking. Equipped with basic knowledge about Wechaty, you can find key problems and use-cases here. If you are a total beginner, go to [Wechaty getting started](https://wechaty.js.org/docs/getting-started/) first.

**Review by proudofsimin — COMMENTED** [review]
I'm a bit confused. If this part is at the very beginning of overview, please includes the prerequisite telling users they need some basic knowledge and direct them to the correct page if they are new to wechaty. 
If this part is not at the beginning, then omit my comment. Thank you.

File: docusaurus/docs/howto/overview.mdx (hunk 1 above)
Comment (abhishek-iiit) [reply docusaurus/docs/howto/overview.mdx]: Thanks @proudofsimin for your helpful feedback :) Please review for the same and have us ready for merge.

**Review by proudofsimin — APPROVED** [review]
Thank you :)

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
ping for merge @huan

File: README.md [hunk 2]
```diff
@@ -120,14 +120,25 @@ At the same time, we also meet up offline all over the world. Here are some acti
...
+
 ### How To Contribute To the Documentation
 
-- Fork this repository
-- Create your documentation branch: `git checkout -b branch-name`
-- Make changes to the documentation in markdown
-- Commit your changes using the command `git commit -am 'commit message'`
-- Push your changes to gitHub
-- Create new Pull Request
+1. Fork this repository
```
Comment (lijiarui) [inline README.md]: Fork this repository: https://github.com/wechaty/wechaty.js.org

File: docusaurus/docs/howto/overview.mdx [hunk 3]
```diff
@@ -1,38 +1,71 @@
...
+
+- [Puppet Mocker](#)
+- [Starter Template](#)
+
+## Deploy on IM platform
+
+Deploy the bot on various instant messaging platform
+
+- [Deploy in WeChat](#)
+- [Deploy in Whatsapp](#)
```
Comment (lijiarui) [inline docusaurus/docs/howto/overview.mdx:40]: We can add:

- Deploy in WeCom
- Deploy in Gitter
- Deploy in Lark
- Deploy in WeChat Official Account
- Deploy in Dingding

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: docusaurus/docs/howto/overview.mdx (hunk 3 above)
Comment (abhishek-iiit) [reply docusaurus/docs/howto/overview.mdx:40]: Thanks a lot for the feedback ,have added the same as suggested :)

File: README.md (hunk 2 above)
Comment (abhishek-iiit) [reply README.md]: added the same as suggested :)

**Pull request closed without merging**